docs/paper/admin/how-to/basic-troubleshooting.md

```diff
@@ -0,0 +1,135 @@
+---
+slug: /basic-troubleshooting
+---
+
+# Basic Troubleshooting
+
+This guide will help you diagnose your server's problem before reporting it to PaperMC or the plugin's author. Please, read this documentation carefully.
+
+:::tip Backup Your Server
+
+People can break their servers while trying to follow this guide. It is highly advised to backup your server before following this guide.  
+It would be ideal to create a test server by copying your production server's file, but that's not always possible.  
+Please, take a backup first.
+
+:::
+
+:::caution Stop Your Server
+
+Before following this guide, stop your server first. Modifying server files while it is still running will corrupt them. This is like having an operation without anesthesia. Your server will scream like you would.
+
+:::
+
+## Read The Error Message
+
+If your server encounters a problem, it will either print an error message on the server console, create a crash report and close itself, or do both.
+
+If your server crashes, the crash report will be saved in the `crash-report` directory.  
+If your server didn't crash, those error messages will be stored in the `log` directory along with other messages.
```

Comment:
The error messages will be stored as part of the logs in the “logs” directory. Note: the logs older than latest will be compressed and not stored as a direct text file.